boards/adafruit/macropad_rp2040/adafruit_macropad_rp2040.dts

```diff
@@ -0,0 +1,256 @@
+/dts-v1/;
+
+#include <raspberrypi/rpi_pico/rp2040.dtsi>
+#include "adafruit_macropad_rp2040-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/led/led.h>
+#include <zephyr/dt-bindings/pwm/pwm.h>
+
+
+/ {
+	model = "Adafruit MacroPad RP2040";
+	compatible = "adafruit,macropad-rp2040";
+
+	chosen {
+		zephyr,console = &uart1;
+		zephyr,shell-uart = &uart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,flash-controller = &ssi;
+		zephyr,code-partition = &code_partition;
+		zephyr,display = &oled;
+	};
+
+	aliases {
+		watchdog0 = &wdt0;
+		led-strip = &ws2812;
+		led0 = &led0;
+	};
+
+	speaker_reg: speaker-reg {
+		compatible = "regulator-fixed";
+		regulator-name = "Speaker";
+		enable-gpios = <&gpio0 14 GPIO_ACTIVE_HIGH>;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		key0: key_0 {
+			gpios = <&gpio0 1 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			label = "Key 0";
```

Comment:
more of a suggestion but I usually avoid these labels, they are not used anywhere, they don't add any more information from what you already have from the node name ("key_0") and it just become one more thing you are going to forget to rename after copy pasting it 9 times, I suggest dropping these from all keys and leds

Reply:
ya, makes sense. I've dropped from keys, left it for the LED as it seems to be common practice for most of the Pico boards (and beyond, really, for better or worse). Thanks for the quick review!